src/components/CardList/CardList.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import Card, { CardProps } from 'components/Card';
+
+export const CardList: React.FC<{ cards: Array<CardProps> }> = ({ cards }) => {
```

Comment:
Лучше выносить типы пропсов наружу
```suggestion
type CardListProps = {
  cards: CardProps[];
};

export const CardList: React.FC<CardListProps> = ({ cards }) => {
```

Reply:
done